homeassistant/components/nuimo.py

```diff
@@ -0,0 +1,108 @@
+"""
+components.nuimo
+~~~~~~~~~~~~~~~~~~
+Component that connects to a Bluetooth reachable Nuimo device.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/nuimo/
+"""
+from logging import getLogger
+import threading
+from homeassistant.const import EVENT_HOMEASSISTANT_STOP
+from homeassistant.helpers import validate_config
+
+REQUIREMENTS = [
+    'https://github.com/shaftoe/nuimo-raspberrypi-demo/archive/'
+    'pip.zip#nuimo_core==0.0.1',
+    'https://github.com/IanHarvey/bluepy/archive/master.zip#bluepy==1.0.3']
```

Comment:
I did fork the upstream project to have a working prototype, I plan to drop it as soon as the needed changes are merged upstream.
About the bluepy dependency, I went through the Pip docs but, if I understand them correctly, it's stated that dependencies set in setup.py must be installable via easy_install, and bluepy is not. Am I missing something?